site/gatsby-site/cypress/e2e/citeEdit.cy.js

```diff
@@ -31,15 +31,14 @@ describe('Edit report', () => {
 
     cy.conditionalIntercept(
       '**/graphql',
-      (req) =>
-        req.body.operationName == 'FindParentIncident' && req.body.variables.report_number == 10,
+      (req) => req.body.operationName == 'FindIncident',
       'findIncident',
       incident
     );
 
     cy.visit(url);
 
-    cy.wait(['@findReportWithTranslations', '@findIncident']);
+    cy.wait(['@findReportWithTranslations', '@findIncident'], { timeout: 30000 });
```

Comment:
Not need for a custom timeout here, it is already 30000

Reply:
Hmm... when I ran it locally it would not pass without that. I think it was defaulting to 8000 for me. 🤷‍♀️

Reply:
[It should be pulling it from here:https://github.com/responsible-ai-collaborative/aiid/blob/65ff4d5ba9574adf7f18b057d70886b8b9e1cb45/site/gatsby-site/cypress.config.js#L7 which is set to `15000` 🤔